packages/static-build/test/fixtures/gatsby-v5/probes.json

```diff
@@ -1,13 +1,13 @@
 {
   "probes": [
-    { "path": "/", "mustContain": "you just made a Gatsby site! 🎉🎉🎉" },
+    { "path": "/", "mustContain": "Gatsby - index" },
     {
       "path": "/ssr/",
-      "mustContain": "This page is <b>rendered server-side</b>"
+      "mustContain": "rendered server-side"
     },
     {
       "path": "/dsg/",
-      "mustContain": "This page is <b>DSG</b>"
```

Comment:
I think its good to keep the html assertions here because its more likely the page rendered correctly. Is especially important to test `/ssr` since thats what SSR is supposed to do.

Reply:
okay

Reply:
sorry I missed this before. added them back now